mesh/settings.py

```diff
@@ -96,8 +96,8 @@
     'default': {
         'ENGINE': 'django.db.backends.mysql', 
         'NAME': 'mesh',
-        'USER': os.getenv("USER"),
-        'PASSWORD': os.getenv("PASS"), 
+        'USER': os.environ.get('DB_USER'),
+        'PASSWORD': os.environ.get('DB_PASS'),
```

Comment:
if this gets merged, can you tell others to change their environment variable name